docs/python/guides/flags.md

```diff
@@ -5,4 +5,238 @@ sidenav: side-nav-python.html
 type: markdown
 ---
 
-## Flags
\ No newline at end of file
+# Flags
+
+`absl.flags` defines a *distributed* command line system, replacing systems like
+`getopt()`, `optparse` and manual argument processing. Rather than an
+application having to define all flags in or near `main()`, each Python module
+defines flags that are useful to it. When one Python module imports another,
+it gains access to the other's flags. (This is implemented by having all modules
+share a common, global registry object containing all the flag information.)
+
+It includes the ability to define flag types (`boolean`, `float`, `integer`,
+`list`), autogeneration of help (in both human and machine readable format) and
+reading arguments from a file. It also includes the ability to automatically
+generate man pages from the help flags.
+
+Flags are defined through the use of one of the `DEFINE_xxx` functions. The
```

Comment:
the use of one of the `DEFINE_xxx` functions. -> use of `DEFINE_*` functions (where the flags' type is used to define the value).